src/base-components/toast-base.js

```diff
@@ -10,8 +11,15 @@ const ToastBase = forwardRef(({
   button,
   startIcon,
   endIcon,
+  width,
   ...rest
 }, ref) => {
+  const styles = makeStyles({
+    root: {
+      width: `${width}`,
+    },
```

Comment:
maybe

```js
root: { width }
```

Reply:
I actually had it just like that but it takes a string with its unit value (px,rem,em,vw). So it wasn't working

Reply:
so it needs to be String? if so `{ width: String(width) }`